index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./style.css">
+    <!-- fontawesome link -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />    <!-- google font link -->
+    <!-- google font link -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,500;0,600;0,700;1,500;1,700&family=Poppins:ital,wght@0,500;0,600;0,700;1,200;1,400;1,500;1,700&family=Roboto:ital,wght@0,100;0,300;0,400;1,100;1,300;1,400&family=Space+Mono&display=swap" rel="stylesheet">
+</head>
+<body>
+    <nav id="nav-bar">
+        <img src="./Images/My Logo.png" alt="logo-img" id="logo-image">
+        <ul id="side-navbar-container">
+            <li>Portfolio</li>
+            <li>About</li>
+            <li>Contact</li>
+        </ul>
+        <img src="./Images/Union.png" alt="" id="union-img">
+    </nav>
+
+    <!-- MAIN SECTION START  -->
+
+    <main>
+        <section id="main-container">
+            <header>
+                <h1 id="main-header">I’m Alexandra.
+                    Glad to see you!</h1>
+            </header>
+    
+            <article>
+                <p> I’m a software developer! I can help you build a product ,
+                    feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded,
+                     don’t hesitate to contact me.
+                </p>
+            </article>
+    
+            <div class="" id="link">
+                LET’S CONNECT
+            </div>
+
+            <div id="social-links">
+                <img src="./Images/twitter-logo.png" class="social-links-img" alt=""> 
+                <span id="linkedin-container">
+                    <img src="./Images/o-png.png" alt="" id="o-png">
+                <img src="./Images/I-png.png" alt="" id="I-png"> 
+                <img src="./Images/linkedin-logo.png" class="social-links-img" alt="">
+                </span>
+                <img src="./Images/M-logo.png" class="social-links-img" alt="">
+                <img src="./Images/github-logo.png" class="social-links-img" alt="">
+                <img src="./Images/peace-sign.png" class="social-links-img" alt="">
+          </div>
```

Comment:
- Hey, great job so far! however, to enforce good use of the html5 semantic tag, the content here is a group of list items, so you should use a `ul` as the container with an `li` to hold each `a` tags that wrap each `img`
```
<ul>
  <li><a href=""><img src="" alt=""></a></li>
</ul>
```

Reply:
Fixed.